src/templates/article.js

```diff
@@ -300,12 +305,6 @@ const ArticlesTemplate = (props) => {
                                     {footer_banner_details && (
```

Comment:
Let's make it one liner without the first bracket as it's just. single component

Reply:
It was just eslint decision, will correct in pr